archetypes/Portfolio/NoEntries.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { CommitActionEnum } from '@tracer-protocol/pools-js';
+import NoItems from '~/public/img/no-queued.svg';
+
+const QUEUED_TYPE_MAP: Record<CommitActionEnum, string> = {
+    [CommitActionEnum.burn]: 'burns',
+    [CommitActionEnum.mint]: 'mints',
+    [CommitActionEnum.flip]: 'flips',
+};
+
+const HISTORY_TYPE_MAP: Record<CommitActionEnum, string> = {
+    [CommitActionEnum.burn]: 'burn',
+    [CommitActionEnum.mint]: 'mint',
+    [CommitActionEnum.flip]: 'flip',
+};
+
+const MAX_COLS = 100;
```

Comment:
should this actually be the largest number of columns among the 3 different types?

Reply:
Nah its just a bit of a hack https://stackoverflow.com/questions/398734/colspan-all-columns